debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/RomClassWalker.java

```diff
@@ -923,15 +896,15 @@ private void allSlotsInStackMapDo(U8Pointer stackMap) throws CorruptDataExceptio
 		 *  - do we want to have a J9ROM_U16_BE type?
 		 *  */
 		classWalkerCallback.addSlot(clazz, SlotType.J9_U16, U16Pointer.cast(cursor), "stackMapFrameCount");
-		frameCount = U16Pointer.cast(cursor).at(0).longValue();
-		frameCount = SWAP2BE((short) frameCount);
+		short frameCount = SWAP2BE(U16Pointer.cast(cursor).at(0).shortValue());
 		cursor = cursor.add(2);
 
+		int stackMapSize = U16.SIZEOF;
```

Comment:
What is the point of stackMapSize? I don't see it used.

Reply:
I think I noticed that, but because it wasn't (yet?) causing a warning I left it alone. I'll remove it.

Reply:
Fixed in 7b0b376af3c1708478cabe255e88abc577c286e9.